sources/chargen.js

```diff
@@ -512,7 +533,27 @@ $(document).ready(function() {
         }
         ctx.drawImage(img, 0, universalSheetHeight+offSetInAdditionToOtherCustomActions);
       } else {
-        drawImage(ctx, img);
+        const assetDirectory = getDirectoryForAsset(filePath);
+        const assetName = getFileNameForAssetPath(filePath);
+
+        for (const [key, value] of Object.entries(base_animations)) {
+          var animationToCheck = key;
+          if (key === "combat_idle") {
+            animationToCheck = "combat";
+          } else if (key === "backslash") {
+            animationToCheck = "1h_slash";
+          } else if (key === "halfslash") {
+            animationToCheck = "1h_halfslash";
```

Comment:
Can we use a map instead of several if/else statements?

Reply:
yes well ideally I would rename 
`1h_slash` to `backslash`
`1h_halfslash` to `halfslash` 
`combat` to `combat_idle`

And then I can remove this ugly if-else. I will do that once we merged this one ok :)